pages/ksphere/kommander/1.2/install-airgapped/index.md

```diff
@@ -49,10 +49,12 @@ Installing air-gapped Kommander does not require any changes in the `inventory.y
 
 # Configure the Kubernetes cluster
 
-The `cluster.yaml` file provides the configuration details for creating your Konvoy cluster. Installing Kommander in an air-gapped environment requires additional configuration. Ensure the `cluster.yaml` contains all the changes outlined in [air-gapped Konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#configure-the-image-registry] documentation. Follow this procedure:
+The `cluster.yaml` file provides the configuration details for creating your Konvoy cluster. Installing Kommander in an air-gapped environment requires extra configuration. Ensure the `cluster.yaml` has all the changes outlined in [air-gapped Konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#configure-the-image-registry] documentation. Follow this procedure:
+
+**TODO**: How is the reader supposed to conduct these following 3 steps?
 
 1.  Ensure Kommander can use the self-hosted charts repository running on top of the Konvoy cluster. It can not connect to the default one through the public Internet.
-1.  Ensure  Kommander can find and access the private Docker registry. The `registry_ip` variable is the IP address of the available private Docker registry. ${user} and ${password} are the username and password if access requires credentials.
+1.  Ensure  Kommander can find and access the private Docker registry. The `bastion_ip` variable in the code snippet below is the IP address of the available private Docker registry. You can omit the username and password lines if your registry does not require authentication.
```

Comment:
We need to replace all the occurences of `bastion_ip` in the test and replace it with `registry_ip`. Otherwise it will be super confusing for our users. The fact that we are using bastion as registry host is just our way of doing things, users can be using totally different approach.

Reply:
Sounds good. Will do that.